Resources/Prototypes/Hydroponics/seeds.yml

```
@@ -355,6 +355,31 @@
       Max: 5
       PotencyDivisor: 5
 
+- type: seed
+  id: teaPlant
```

Since this is a new addition, we have an opportunity to use our new entity name conventions.
```suggestion
  id: TeaPlant
```
You may have to update uses of this ID if you change this.